capstone/src/main/java/com/google/sps/servlets/ChallengeServlet.java

```diff
@@ -40,14 +47,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String challengeName = request.getParameter("name");
     LocalDateTime dueDate = this.getDueDate(LocalDateTime.now());
     long dueDateMillis = Timestamp.valueOf(dueDate).getTime();
-    Challenge challenge = new Challenge(challengeName, dueDateMillis, null,
-        new ArrayList<String>());
+    Challenge challenge =
+        new Challenge(challengeName, dueDateMillis, null, new ArrayList<String>());
```

Comment:
you could include an inline comment before the null parameter mentioning what it is